lib/plug/adapters/cowboy.ex

```diff
@@ -31,6 +31,9 @@ defmodule Plug.Adapters.Cowboy do
     * `:timeout` - Time in ms with no requests before Cowboy closes the connection.
       Defaults to 5000ms.
 
+    * `:no_default_onresponse` - Disable the default `onresponse/4` callback handler.
```

Comment:
I think this name is based on the implementation and not on the feature. We should probably call it `:log_error_on_incomplete_requests` (the default being true).

Reply:
Cool by me, I really wasn't sure what to call this option. PR updated.